cypress.config.ts

```diff
@@ -7,10 +7,10 @@ import caseloads from './integration_tests/mockApis/caseloads'
 import courses from './integration_tests/mockApis/courses'
 import courseParticipations from './integration_tests/mockApis/courseParticipations'
 import manageUsers from './integration_tests/mockApis/manageUsers'
-import prisons from './integration_tests/mockApis/prisons'
-import prisoners from './integration_tests/mockApis/prisoners'
+import prison from './integration_tests/mockApis/prison'
+import prisonRegister from './integration_tests/mockApis/prisonRegister'
```

Comment:
These two should probably have `Api` at the end for consistency with the rest of the codebase? I think we should separately remove the "API" from the Prison Register to use its proper name (Prison API, Prison Register, and Prisoner Search appear to be the proper names?)